### ISSUE TITLE
Can't manage to pass the mutmut test

### ISSUE THREAD

**Issue by Saxamos** [boxed/mutmut#12]
Hi, I'm trying your lib but I can't manage to pass the mutmut test, here is my code : 
```
def addition(a, b):
    return a < b
```

and the associated test : 
```

def test_addition():
    a = 1
    b = 2

    result = addition(a, b)

    assert result == True


def test_addition_2():
    a = 1
    b = 1

    result = addition(a, b)

    assert result == False
```

and my command : 
```
mutmut app/add.py
mutmut tests/
```

Is there anything wrong ?

Thanks,
Samuel

**Comment by boxed** [comment]
Yea. It would help if you had given the output or errors you got, but I'm going to guess based on what you supplied...

The command to mutmut is a list of directories to mutate, not files. So the command is

```
mutmut app
```

in your case. Mutmut tries to guess a bit so if the parent directory is also called the same, it can find it itself, also "src" and "lib" dirs are found automatically.

I've attached a zip with how I interpreted your project: [mutmut_issue_12.zip](https://github.com/boxed/mutmut/files/1356913/mutmut_issue_12.zip) When running the command above in this project I get this output:

```
(venv) ➜  mutmut_issue_12 mutmut app/
--- starting mutation ---
1 out of 1  (file: app/app.py, mutation: 0)
```

Which isn't terribly interesting but the example code you have only has one mutation point and the tests cover it pretty nicely :P
